database/seed.js

```diff
@@ -0,0 +1,64 @@
+const data = require('./data.js');
+const mongoose = require('mongoose');
+
+mongoose.set('useNewUrlParser', true);
+mongoose.set('useFindAndModify', false);
+mongoose.set('useCreateIndex', true);
+mongoose.set('useUnifiedTopology', true);
+mongoose.connect('mongodb://localhost/moovies');
+
+
+const mooviesSchema = mongoose.Schema({
+  title: String,
+  trailer_title: String,
+  description: String,
+  running_time: Number,
+  date: Date,
+  ratings: Number,
+  thumbnail_url: String,
+  video_url: String,
+});
+
+const Moovie = mongoose.model('Moovie', mooviesSchema);
+
+const seed = function(arr, callback) {
+  let data = arr.map(function(item) {
+    return new Moovie({
+      title: item.title,
+      description: item.description,
+      running_time: item.running_time,
+      date: item.date,
+      ratings: item.ratings,
+      thumbnail_url: item.thumbnail_url,
+      video_url: item.video_url
+    })
+  })
+
+  Moovie.deleteMany({}, (err) => {
```

Comment:
Are you deleting all the records in the collection and inserting the videos every time you seed?

Reply:
Hi Brian, exactly!